dvc/remote/base.py

```diff
@@ -552,8 +552,8 @@ def upload(self, from_info, to_info, name=None, no_progress_bar=False):
         if not hasattr(self, "_upload"):
             raise RemoteActionNotImplemented("upload", self.scheme)
 
-        if to_info.scheme != self.scheme:
-            raise NotImplementedError
+        # if to_info.scheme != self.scheme:
+        #     raise NotImplementedError
```

Comment:
I suppose this PR is WIP? In that case, please add `[WIP]` prefix :slightly_smiling_face: These lines shouldn't be removed in the final PR, unless there is a really good reason for it.

Reply:
If I understand correctly, to determine which 'remote' to use - the scheme is used. For this, I added the 'webdav' scheme, but it works on http. But the base is checking for compliance with the scheme - and removed it.